### PR TITLE
created day2 for terraform-with-aws i.e local-state-file(s3,lambda,be…

### PR DESCRIPTION
**Pull request by DivyaMandati**
created day2 for terraform-with-aws i.e local-state-file(s3,lambda,beanstalk)

### CODE REVIEW

File: terraform-with-aws/day2/2a-local-state-file/variables.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+locals {
```
Comment (techslateramu) [inline terraform-with-aws/day2/2a-local-state-file/variables.tf:1]: please move them to locals.tf

File: terraform-with-aws/day2/2a-local-state-file/main.tf [hunk 2]
```diff
@@ -0,0 +1,78 @@
+resource "aws_s3_bucket" "bucket" {
+  bucket        = local.bucket_name
+  force_destroy = true
```
Comment (techslateramu) [inline terraform-with-aws/day2/2a-local-state-file/main.tf:3]: please move this to variable.... don't hard code !

File: terraform-with-aws/day2/2a-local-state-file/main.tf [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+resource "aws_s3_bucket_public_access_block" "bucket" {
+  bucket = aws_s3_bucket.bucket.id
+
+  block_public_acls       = true
+  block_public_policy     = true
```
Comment (techslateramu) [inline terraform-with-aws/day2/2a-local-state-file/main.tf:10]: Please dont hard code them, move them to variables.

**Review by techslateramu — CHANGES_REQUESTED** [review]
Please fix all the review comments !

**Comment by DivyaMandati** [comment]
fixed all the requested issue please have a look

**Review by techslateramu — APPROVED** [review]

**Pull request merged**